src/amo/components/AddonReviewList/styles.scss

```diff
@@ -1,57 +1,63 @@
-@import "~amo/css/inc/vars";
 @import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
 
 .AddonReviewList {
-  padding: $padding-page;
+  margin: $padding-page;
 
   @include respond-to(large) {
-    padding: $padding-page-l;
+    display: flex;
+    margin: $padding-page-l;
   }
 
   .CardList {
-    margin-bottom: $vertical-space-xs;
+    margin: 0 0 $vertical-space-xs;
+  }
+}
+
+.AddonReviewList-addon {
+  @include respond-to(large) {
+    min-width: 300px;
+    width: 35%;
+  }
+}
+
+.AddonReviewList-reviews {
+  @include respond-to(large) {
+    @include margin-start(12px);
+
+    width: 65%;
   }
 }
 
 .AddonReviewList-header {
-  align-items: center;
   display: flex;
-  margin: 15px 0;
-
-  .AddonReviewList-header-text {
-    // Flex implicitly sets `min-width: auto`; to force the overflow fix
-    // for https://github.com/mozilla/addons-frontend/issues/2415
-    // we set a min-width.
-    min-width: 1%;
-
-    > h2,
-    > h3 {
-      font-size: 1.1em;
-    }
-
-    > h2 {
-      margin: 0 0 5px;
-      padding-top: 10px;
-    }
-
-    > h3 {
-      font-weight: normal;
-      margin: 0;
-    }
-
-    a:link,
-    a:visited,
-    a:hover,
-    a:active {
-      color: $primary-font-color;
-      font-weight: normal;
-      text-decoration: none;
-      word-wrap: break-word;
-    }
+  padding: 12px 0;
+}
+
+.AddonReviewList-header-icon {
+  @include margin-end(18px);
+}
+
+.AddonReviewList-header-addonName {
```

Comment:
I dont know how to do it but ideally, the text next to the icon should be vertically centered.

<img width="1392" alt="screen shot 2017-10-24 at 15 03 31" src="https://user-images.githubusercontent.com/217628/31944551-bd0d422a-b8cc-11e7-9963-e0f95b453a2c.png">

Reply:
Frustratingly the mocks have much larger icons than I think we actually have, so they'll look pretty blurry. I'll give that a go though.

Reply:
Looks like this is not solved, right?